diff -urp /etc/pam.d/sshd /etc/pam.d/sshd
--- /etc/pam.d/sshd	2018-01-18 22:16:00.000000000 +0800
+++ /etc/pam.d/sshd	2018-03-10 22:10:04.322399294 +0800
@@ -19,7 +19,7 @@ account    required     pam_nologin.so
 session [success=ok ignore=ignore module_unknown=ignore default=bad]        pam_selinux.so close
 
 # Set the loginuid process attribute.
-session    required     pam_loginuid.so
+session    optional     pam_loginuid.so
 
 # Create a new session keyring.
 session    optional     pam_keyinit.so force revoke
diff -urp /etc/profile /etc/profile
--- /etc/profile	2015-10-23 01:15:21.000000000 +0800
+++ /etc/profile	2018-03-10 22:10:04.322399294 +0800
@@ -25,3 +25,5 @@ if [ -d /etc/profile.d ]; then
   done
   unset i
 fi
+
+export VISIBLE=now
diff -urp /etc/ssh/sshd_config /etc/ssh/sshd_config
--- /etc/ssh/sshd_config	2018-03-14 00:44:38.000000000 +0800
+++ /etc/ssh/sshd_config	2018-03-10 22:10:04.322399294 +0800
@@ -25,7 +25,7 @@ LogLevel INFO
 
 # Authentication:
 LoginGraceTime 120
-PermitRootLogin prohibit-password
+PermitRootLogin yes
 StrictModes yes
 
 RSAAuthentication yes
